contract_mandate/__manifest__.py

```diff
@@ -5,7 +5,7 @@
 {
     'name': 'Contract Mandate',
     'summary': 'Mandate in contracts and their invoices',
-    'version': '9.0.1.0.0',
+    'version': '10.0.1.0.0',
     'author': 'Tecnativa, '
               'Odoo Community Association (OCA)',
     'website': 'http://www.tecnativa.com',
```

Comment:
@carlosdauden URL should be `https://github.com/OCA/contract`

Reply:
@nikul-serpentcs this PR was done in 2017 when that rule doesn't exists.

Reply:
> @nikul-serpentcs this PR was done in 2017 when that rule doesn't exists.

Ok, No Problem @rafaelbn :+1: